### PR TITLE
Update excel-quickstart-vue.md

### PR DESCRIPTION
**Pull request by lindalu-MSFT**
Fixed missing colons, contractions, procedure order (imperative), global English, styleguide stuff

### CODE REVIEW

File: docs/quickstarts/excel-quickstart-vue.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ ms.prod: excel
 localization_priority: Priority
 ---
 
-# Build an Excel task pane add-in using Vue
+# Use Vue to build an Excel task pane add-in
```
Comment (AlexJerabek) [inline docs/quickstarts/excel-quickstart-vue.md:9]: The existing phrasing matches the other quick starts. Let's leave as is unless we plan on changing everything at once.

File: docs/quickstarts/excel-quickstart-vue.md [hunk 2]
```diff
@@ -58,7 +58,7 @@ Each add-in requires a manifest file to define its settings and capabilities.
 
     ![Screenshot of the Yeoman Office Add-in generator command line interface, with project type set to manifest only.](../images/yo-office-manifest-only-vue.png)
 
-After you complete the wizard, it creates a `My Office Add-in` folder, which contains a `manifest.xml` file. You will use the manifest to sideload and test your add-in at the end of the quick start.
+After completion, the wizard creates a `My Office Add-in` folder containing a `manifest.xml` file. You'll use the manifest to sideload and test your add-in.
```
Comment (AlexJerabek) [inline docs/quickstarts/excel-quickstart-vue.md:61]: Question: is the wizard just the user questions or the installation as well? Wikipedia is sending mixed messages: https://en.wikipedia.org/wiki/Wizard_(software)

File: docs/quickstarts/excel-quickstart-vue.md [hunk 3]
```diff
@@ -181,29 +181,29 @@ After you complete the wizard, it creates a `My Office Add-in` folder, which con
...
 
-2. In Excel, choose the **Home** tab, and then choose the **Show Taskpane** button in the ribbon to open the add-in task pane.
+2. Open the add-in task pane in Excel. On the **Home** tab, choose the **Show Taskpane** button.
 
    ![Screenshot of the Excel Home menu, with the Show Taskpane button highlighted.](../images/excel-quickstart-addin-2a.png)
 
 3. Select any range of cells in the worksheet.
 
-4. In the task pane, choose the **Set color** button to set the color of the selected range to green.
+4. Set the color of the selected range to green. In the task pane, choose the **Set color** button.
```
Comment (AlexJerabek) [inline docs/quickstarts/excel-quickstart-vue.md:206]: Not related, but let's make it more clear which task pane the user needs to go to.
```suggestion
4. Set the color of the selected range to green. In your add-in's task pane, choose the **Set color** button.
```

Comment (lindalu-MSFT) [reply docs/quickstarts/excel-quickstart-vue.md:206]: The illustration following this step makes it clear.

File: docs/quickstarts/excel-quickstart-vue.md (hunk 2 above)
Comment (lindalu-MSFT) [reply docs/quickstarts/excel-quickstart-vue.md:61]: Has anyone besides me tested this? When I tried, the wizard created a folder with the manifest in it.

File: docs/quickstarts/excel-quickstart-vue.md (hunk 1 above)
Comment (lindalu-MSFT) [reply docs/quickstarts/excel-quickstart-vue.md:9]: I fixed the other topics as well and committed to this same PR. Better for SEO, accessibility, and more compliant with MSFT style.

**Comment by lindalu-MSFT** [comment]
@AlexJerabek I addressed the issues, please merge this PR.

File: docs/quickstarts/excel-quickstart-vue.md [hunk 4]
```diff
@@ -15,21 +15,21 @@ In this article, you'll walk through the process of building an Excel task pane
 [!include[Set up requirements](../includes/set-up-dev-environment-beforehand.md)]
 [!include[Yeoman generator prerequisites](../includes/quickstart-yo-prerequisites.md)]
 
-- Install the [Vue CLI](https://cli.vuejs.org/) globally.
+- Install the [Vue CLI](https://cli.vuejs.org/) globally. From the terminal, run the following command:
```
Comment (ElizabethSamuel-MSFT) [inline docs/quickstarts/excel-quickstart-vue.md:18]:
```suggestion
- Install the [Vue CLI](https://cli.vuejs.org/) globally. From the terminal, run the following command.
```

File: docs/quickstarts/excel-quickstart-vue.md [hunk 5]
```diff
@@ -67,7 +67,7 @@ After you complete the wizard, it creates a `My Office Add-in` folder, which con
 
 [!include[HTTPS guidance](../includes/https-guidance.md)]
 
-1. To enable HTTPS for your app, create a `vue.config.js` file in the root folder of the Vue project with the following contents:
+1. Enable HTTPS for your app. In the root folder of the Vue project, create a `vue.config.js` file with the following contents:
```
Comment (ElizabethSamuel-MSFT) [inline docs/quickstarts/excel-quickstart-vue.md:70]:
```suggestion
1. Enable HTTPS for your app. In the root folder of the Vue project, create a `vue.config.js` file with the following contents.
```

**Review by ElizabethSamuel-MSFT — COMMENTED** [review]
Per our discussion about "following" and periods vs colons.

File: docs/quickstarts/excel-quickstart-vue.md (hunk 2 above)
Comment (AlexJerabek) [reply docs/quickstarts/excel-quickstart-vue.md:61]: @lindalu-MSFT  The tool does create the folders. My question is more about the pedantic discussion between the wizard and the tool. It as my understanding that a wizard is only the part of setup that prompts the user, not the part of the installer that actually creates files and folders.

File: docs/quickstarts/excel-quickstart-angular.md [hunk 6]
```diff
@@ -1,12 +1,12 @@
 ---
-title: Build an Excel task pane add-in using Angular
+title: USe Angular to build an Excel task pane add-in
```
Comment (ElizabethSamuel-MSFT) [inline docs/quickstarts/excel-quickstart-angular.md:2]:
```suggestion
title: Use Angular to build an Excel task pane add-in
```

File: docs/quickstarts/onenote-quickstart.md [hunk 7]
```diff
@@ -100,8 +100,7 @@ try {
     - If you're signed in with your work or education account, select the **MY ORGANIZATION** tab, and then select **Upload My Add-in**.
 
     The following image shows the **MY ADD-INS** tab for consumer notebooks.
-
-    <img alt="Screenshot of the Office Add-ins dialog showing the MY ADD-INS tab" src="../images/onenote-office-add-ins-dialog.png" width="500">
+    ![Screenshot of the Office Add-ins dialog showing the MY ADD-INS tab.](../images/onenote-office-add-ins-dialog.png)
```
Comment (ElizabethSamuel-MSFT) [inline docs/quickstarts/onenote-quickstart.md:103]:
```suggestion

    ![Screenshot of the Office Add-ins dialog showing the MY ADD-INS tab.](../images/onenote-office-add-ins-dialog.png)
```

**Review by ElizabethSamuel-MSFT — APPROVED** [review]
Looks good - just a couple of suggested tweaks.

**Pull request merged**